packages/client/src/definitions/LinkDefinitions.ts

```diff
@@ -39,7 +35,7 @@ export type OsdkObjectLinksEntry<
   O extends ObjectTypeDefinition<any>,
   L extends ObjectTypeLinkKeysFrom2<O>,
 > = O["links"][L] extends ObjectTypeLinkDefinition<infer T, infer M> ? (
-    M extends false ? SingleLinkAccessor<T> : MultiLinkAccessor<T>
+    M extends false ? SingleLinkAccessor<T> : BaseObjectSet<T>
```

Comment:
Should this just be `ObjectSet<T>`?

Reply:
BaseObjectSet also has the .get(pk) method.